testutil/config/example_migration.md

```diff
@@ -0,0 +1,119 @@
+# Chain Configuration Migration Guide
```

Comment:
The title of this file is pretty generic.  How are we pointing people to this file?

Feels like it needs to be linked in the changelog or in an overall "migration guide" 

we have stuff [here](https://github.com/cosmos/evm/tree/main/docs/migrations) for previous releases